test/test_stac.py

```diff
@@ -110,9 +110,90 @@ def test_get_item_collection_searchable_catalog(self):
             "sentinel-2-l2a/S2A_32UNU_20200302_0_L2A"
         ]
         self.assertIsInstance(items, ItemCollection)
-        self.assertCountEqual(data_id_items, data_id_items_expected)
+        self.assertCountEqual(data_id_items_expected, data_id_items)
         self.assertEqual(len(items), len(data_id_items))
 
+    @pytest.mark.vcr()
+    def test_get_data_ids(self):
+        stac_instance = Stac(self.url_nonsearchable)
+        data_ids = stac_instance.get_data_ids()
+        data_ids_expected = [
+            "zanzibar-collection/znz001/raster",
+            "zanzibar-collection/znz029/raster",
+            "spacenet-buildings-collection/AOI_2_Vegas_img2636/raster",
+            "spacenet-buildings-collection/AOI_3_Paris_img1648/raster",
+            "spacenet-buildings-collection/AOI_4_Shanghai_img3344/raster"
+        ]
+        for (data_id, data_id_expected) in zip(data_ids, data_ids_expected):
+            self.assertEqual(data_id_expected, data_id)
```

Comment:
But `data_id` is an iterator. Does this still work?

Reply:
Yes, I tried it! See also [this PR](https://github.com/python/cpython/pull/686). I agree that it's not obvious, though -- the main Python unittest docs aren't generated from the (correct) docstrings and refer to "sequence" rather than "iterable".